src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigClientImplTest.java

```diff
@@ -107,6 +109,24 @@ public void testGetTemplate() throws Exception {
     assertTrue(
             headerParameter.getDefaultValue() instanceof ParameterValue.InAppDefault);
     checkGetRequestHeader(interceptor.getLastRequest());
+
+    // Check Conditions
+    List<Condition> actualConditions = template.getConditions();
+    List<Condition> expectedConditions = new ArrayList<>();
+    expectedConditions
+            .add(new Condition("ios_en", "device.os == 'ios' && device.country in ['us', 'uk']")
+                    .setTagColor(TagColor.INDIGO));
+    expectedConditions
+            .add(new Condition("android_en",
+                    "device.os == 'android' && device.country in ['us', 'uk']")
+                    .setTagColor(TagColor.UNSPECIFIED));
+    assertEquals(expectedConditions.size(), actualConditions.size());
+    for (int i = 0; i < expectedConditions.size(); i++) {
+      assertEquals(expectedConditions.get(i).getName(), actualConditions.get(i).getName());
```

Comment:
Might be meaningful override `equals()` in `Condition` class, and use that here.

Reply:
Good call! Thanks!